src/Venflow/Venflow/DatabaseConfigurationOptionsBuilder.cs

```diff
@@ -62,6 +65,19 @@ public DatabaseConfigurationOptionsBuilder UseConfigurations(params Assembly[] a
             return this;
         }
 
+        /// <summary>
+        /// Sets the naming convention to be used for entity table & column names.
+        /// </summary>
+        /// <typeparam name="T">An implementation of <see cref="INpgsqlNameTranslator"/> to be used for name translation.</typeparam>
+        /// <returns>The same builder instance so that multiple calls can be chained.</returns>
+        public DatabaseConfigurationOptionsBuilder SetNamingConvention<T>()
+            where T : INpgsqlNameTranslator, new()
+        {
+            NpgsqlNameTranslator = new T();
+
+            return this;
+        }
+
```

Comment:
I'd also add an overload which accepts an instance of type `INpgsqlNameTranslator` as even some translators built into Npgsql have constructor parameters, see [here](https://www.npgsql.org/doc/api/Npgsql.NameTranslation.NpgsqlSnakeCaseNameTranslator.html#Npgsql_NameTranslation_NpgsqlSnakeCaseNameTranslator__ctor_System_Boolean_).